.travis.yml

```diff
@@ -1,5 +1,13 @@
 sudo: false
 language: java
-jdk: 
+env:
+  global:
+    - AWS_ACCESS_KEY_ID=AKIAJIZZTZIXJIFQQ2WA
+    - secure: "b1v74SOUrdIn/KkPFJ+2UZ8u9qoJxahwEA7yjUWT+XIYQIipzXKUfJvuTPtmKGdhxnaqY5zI7C9rzE775pKBNG4IGMmRwRvmoxatc7UjunQ7o1jre8MnitPT3aMIZU5flXb/gJ3if5RHoVpAeDtT1FUMgt1pZtVjh0ldcrh9oRc="
```

Comment:
It is, insofar as we trust travis.  It's their conventional way of
encrypting secrets safely for inclusion in .travis.yml.  The keypair
belongs to a user whose only rights are uploading to the code-deploys
bucket.

On Tue, Jul 12, 2016 at 11:31 AM, Tom G Varik <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/casetext/fnv-text-profile-signature/pull/3#discussion_r70495103>
> :
>
> > @@ -1,5 +1,13 @@
> >  sudo: false
> >  language: java
> > -jdk:
> > +env:
> > +  global:
> > +    - AWS_ACCESS_KEY_ID=AKIAJIZZTZIXJIFQQ2WA
> > +    - secure: "b1v74SOUrdIn/KkPFJ+2UZ8u9qoJxahwEA7yjUWT+XIYQIipzXKUfJvuTPtmKGdhxnaqY5zI7C9rzE775pKBNG4IGMmRwRvmoxatc7UjunQ7o1jre8MnitPT3aMIZU5flXb/gJ3if5RHoVpAeDtT1FUMgt1pZtVjh0ldcrh9oRc="
>
> I'm not familiar with this. I assume it's setting AWS_SECRET_ACCESS_KEY.
> Is it safe to commit this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/casetext/fnv-text-profile-signature/pull/3/files/20f3109267bd56bb764c4e2cdd85b2bd6e6d5ced#r70495103>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFmhf8ScqjDwu02Z9oKxxSb9eaA3nznlks5qU92EgaJpZM4JKroL>
> .
>